admin/class-boldgrid-backup-admin-log.php

```diff
@@ -147,6 +150,72 @@ public function add_last_error() {
 		$this->last_error = $current_error;
 	}
 
+	/**
+	 * Add user-friendly messaging to the log regarding the last error
+	 *
+	 * @param array $current_error The error array to format.
+	 * @since 1.5.7
```

Comment:
Need to change the order. Please see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods

Reply:
Example changes:

```
	/**
	 * Add user-friendly messaging to the log regarding the last error
	 *
	 * @since 1.5.7
	 *
	 * @param array $current_error The error array to format.
	 */
```

Reply:
fixed and added return to doc block